src/shell-integration/elvish/lib/ghostty-integration.elv

```diff
@@ -75,7 +75,8 @@
   }
 
   fn report-pwd {
-    printf "\e]7;file://%s%s\a" (hostname) (pwd)
+    use platform
+    printf "\e]7;file://%s%s\a" platform:hostname $pwd
```

Comment:
Separately, OSC 7 with the `file://` scheme expects the URL to be percent-encoded. If there isn't a convenient way to do that in elvish, we should switch this to the `kitty-shell-cwd://` scheme, which does not use percent encoding.

Reply:
#7033 makes this improvement